src/test/reducers/getBookmarks.test.js

```diff
@@ -0,0 +1,80 @@
+import getBookmarkReducer from '../../reducers/getBookmarkReducer';
+import { getBookmarkType } from '../../actions/getBookmark';
+import { mockState } from '../setup/index';
+
+const { bookmark } = mockState;
+console.log('----', bookmark);
```

Comment:
**Nitpick**

- No `console.log`, houndCI must have flagged this. Has it been resolved @fejiroofficial ?

Reply:
@devdbrandy resolved